tests/rptest/tests/debug_bundle_test.py

```diff
@@ -82,11 +82,16 @@ def test_post_debug_bundle(self, ignore_none: bool):
         assert res.status_code == requests.codes.ok, res.json()
         assert res.json()['status'] == 'success', res.json()
         assert res.json()['job_id'] == str(job_id), res.json()
-        assert res.json()['filename'] == f"{job_id}.zip", res.json()
+        filename = res.json()['filename']
+        assert filename == f"{job_id}.zip", res.json()
 
         # Delete the debug bundle after it has completed, expect a conflict
         try:
             admin.delete_debug_bundle(job_id=job_id, node=node)
         except requests.HTTPError as e:
             assert e.response.status_code == requests.codes.conflict, res.json(
             )
+
+        res = admin.get_debug_bundle_file(filename=filename, node=node)
+        assert res.status_code == requests.codes.ok, res.json()
+        assert res.headers['Content-Type'] == 'application/zip', res.json()
```

Comment:
chore/question: can this be compared with the file that was generated on disk?

Reply:
Compared sha256sum